crates/re_types/src/datatypes/mesh_properties.rs

```diff
@@ -94,88 +96,54 @@ impl ::re_types_core::Loggable for MeshProperties {
-                                    .map(|datum| datum.num_instances())
-                                    .unwrap_or_default()
-                            }),
-                        )
-                        .unwrap()
-                        .into();
-                        ListArray::new(
-                            DataType::List(Box::new(Field {
-                                name: "item".to_owned(),
-                                data_type: DataType::UInt32,
-                                is_nullable: false,
-                                metadata: [].into(),
-                            })),
-                            offsets,
-                            PrimitiveArray::new(
-                                DataType::UInt32,
-                                indices_inner_data,
-                                indices_inner_bitmap,
-                            )
-                            .boxed(),
-                            indices_bitmap,
-                        )
-                        .boxed()
-                    }
-                }],
-                bitmap,
-            )
-            .boxed()
+                    <crate::datatypes::MeshProperties>::arrow_datatype(),
+                    vec![
+                        { let (somes, indices) : (Vec < _ >, Vec < _ >) = data.iter()
+                        .map(| datum | { let datum = datum.as_ref().map(| datum | { let
+                        Self { indices, .. } = & * * datum; indices.clone() }).flatten();
+                        (datum.is_some(), datum) }).unzip(); let indices_bitmap : Option
+                        < ::re_types_core::external::arrow2::bitmap::Bitmap > = { let
+                        any_nones = somes.iter().any(| some | ! * some); any_nones
+                        .then(|| somes.into()) }; { use
+                        ::re_types_core::external::arrow2:: { buffer::Buffer,
+                        offset::OffsetsBuffer }; let indices_inner_data : Buffer < _ > =
+                        indices.iter().flatten().map(| b | b.as_slice()).collect:: < Vec
+                        < _ >> ().concat().into(); let indices_inner_bitmap : Option <
+                        ::re_types_core::external::arrow2::bitmap::Bitmap > = None; let
+                        offsets = ::re_types_core::external::arrow2::offset::Offsets:: <
+                        i32 > ::try_from_lengths(indices.iter().map(| opt | opt.as_ref()
+                        .map(| datum | datum.num_instances()).unwrap_or_default()))
+                        .unwrap().into(); ListArray::new(DataType::List(Box::new(Field {
+                        name : "item".to_owned(), data_type : DataType::UInt32,
+                        is_nullable : false, metadata : [].into(), })), offsets,
+                        PrimitiveArray::new(DataType::UInt32, indices_inner_data,
+                        indices_inner_bitmap,).boxed(), indices_bitmap,).boxed() } },
+                    ],
```

Comment:
let's try to solve this problem, because this is 😬

Reply:
there we go